### PR TITLE
Update tls.md

### LINKED ISSUE #2743: [DOC] Configure TLS topic for OpenSearch Dashboards is misleading
**What do you want to do?**
 
- [X] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
I am using version 2.5.0 of both OpenSearch and OpenSearch Dashboards.

I have recently tried configuring OpenSearch Dashboards to connect to an OpenSearch instance that we have secured with TLS encryption on all ports. 

I was referring to this documentation: https://opensearch.org/docs/latest/install-and-configure/install-dashboards/tls/

This topic provides insufficient information on the settings available within OpenSearch Dashboards. I was trying to specify the client cert and key via `server.ssl.certificate` and `server.ssl.key` since those are the settings given in the table. This led to a frustrating time watching the connection continually fail with `Socket hang up` errors.

It was only after I started picking through the entrypoint script for the OpenSearch Dashboards container that I found the settings I actually needed, which are `opensearch.ssl.certificate` and `opensearch.ssl.key`. I also discovered there are many more settings that simply are not documented.

Ideally a more comprehensive set of reference documentation that covers all the settings available and how to use them would exist (maybe I just can't find it!).

Let me know if there's any further information I can provide to help clarify this issue.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
N/A

### PR DESCRIPTION
**Pull request by mitct23**
Update TLS parameters

### Description
Add required parameters when mTLS is enforced in OpenSearch.

### Fixes

Fixes #2743 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _install-and-configure/install-dashboards/tls.md
Comment (vagimeli) [inline _install-and-configure/install-dashboards/tls.md:16]:
```suggestion
server.ssl.enabled | Allows for communication between OpenSearch Dashboards and the web browser. Set to `true` for HTTPS or `false` for HTTP.
```

File: _install-and-configure/install-dashboards/tls.md
Comment (vagimeli) [inline _install-and-configure/install-dashboards/tls.md:17]:
```suggestion
server.ssl.certificate | If `server.ssl.enabled` is set to `true`, specify the full path to a valid server certificate for OpenSearch Dashboards. You can [generate your own certificate]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority.
```

**Comment by hdhalter** [comment]
Hi @mitct23 , thanks so much for submitting a PR to the documentation repo! Is this ready for review or are you still making updates? Also, did you find this problem yourself? We have a couple open issues on this topic which could be satisfied by this update: https://github.com/opensearch-project/documentation-website/issues/2743 and the 4th item in https://github.com/opensearch-project/documentation-website/issues/5420.

File: _install-and-configure/install-dashboards/tls.md
Comment (vagimeli) [inline _install-and-configure/install-dashboards/tls.md:18]:
```suggestion
server.ssl.key | If `server.ssl.enabled` is set to `true`, specify the full path, for example, `/usr/share/opensearch-dashboards-1.0.0/config/my-client-cert-key.pem`, to the key for your server certificate. You can [generate your own certificate]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority.
```

**Review by vagimeli — COMMENTED** [review]
Document review completed. Changes requested before this doc is merged.

File: _install-and-configure/install-dashboards/tls.md
Comment (mitct23) [reply _install-and-configure/install-dashboards/tls.md]: Updated

**Comment by hdhalter** [comment]
@vagimeli - Who is doing the tech review for this? 
@peterzhuamazon - Can you take a look at this, please? Thanks!

**Comment by gaiksaya** [comment]
Adding security dashboards plugin maintainers for review: @cwperks @DarshitChanpura @scrawfor99

File: _install-and-configure/install-dashboards/tls.md
Comment (scrawfor99) [inline _install-and-configure/install-dashboards/tls.md:35]: I believe both this and the setting below are currently supported. Not sure why both are implemented but I in the meantime we should probably keep both documented.

File: _install-and-configure/install-dashboards/tls.md
Comment (scrawfor99) [inline _install-and-configure/install-dashboards/tls.md:13]: There are some other settings you can find here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/28c1fbb84550d33a036e231661867988af5ebe54/src/cli/serve/serve.js#L102

Which may be worth adding to this doc if not present elsewhere.

**Review by scrawfor99 — COMMENTED** [review]
Thanks a ton!

File: _install-and-configure/install-dashboards/tls.md
Comment (mitct23) [reply _install-and-configure/install-dashboards/tls.md]: updated.

File: _install-and-configure/install-dashboards/tls.md
Comment (mitct23) [reply _install-and-configure/install-dashboards/tls.md:35]: opensearch.ssl.certificateAuthorities - CA who signed the server certificate used in OpenSearch cluster
server.ssl.certificateAuthorities - CA who signed the client certificate if mTLS is enabled for Dashboard

File: _install-and-configure/install-dashboards/tls.md
Comment (mitct23) [reply _install-and-configure/install-dashboards/tls.md:13]: added info about more parameters

**Review by vagimeli — APPROVED** [review]
@scrawfor99 LGTM

**Review by vagimeli — APPROVED** [review]
Doc review complete. See changes for clarity and writing style. I've moved this to editorial review.

**Review by natebower — COMMENTED** [review]
@vagimeli Please see my comments and changes and let me know if you have any questions. Thanks!

**Review by DarshitChanpura — APPROVED** [review]
LGTM from security settings perspective.

**Comment by vagimeli** [comment]
> @vagimeli Please see my comments and changes and let me know if you have any questions. Thanks!

@natebower I've committed your edits and addressed the comments. I defined abbreviations on first mention. Thanks for the thorough review!

**Pull request merged**
